loadavg for Windows is not supported

**Describe the bug**

 When i try using egress in local development, i got error message **egress not connected (redis required)**, so i think server didn't connect to redis automatically and then i try to load my own config file, but i got error too. I didn't have any idea to test 

**Server**
- Version: 1.3.1
- Environment: local dev
- config file changed : 
 - `use_extrnal_ip` set to `false`

Screenshot
![image](https://user-images.githubusercontent.com/11814324/202780858-ab3c19ba-0c85-459b-8bcd-bb272d490c30.png)

**Command**
```bash
server --dev --config config-development.yml
```

**Output:**
2022-11-19T18:25:42.300+0700    INFO    livekit service/wire_gen.go:186 using multi-node routing via redis      {"sentinel": false, "addr": "localhost:6379"}
2022-11-19T18:25:42.733+0700    DEBUG   livekit routing/redisrouter.go:377      starting redisWorker    {"nodeID": "ND_LIFp2Z23"}
2022-11-19T18:25:43.356+0700    INFO    livekit service/server.go:212   starting LiveKit server {"portHttp": 7880, "nodeID": "ND_LIFp2Z23", "nodeIP": "169.254.7.197", "version": "1.3.1", "rtc.portTCP": 7881, "rtc.portICERange": [50000, 60000]}
2022-11-19T18:25:44.746+0700    ERROR   livekit routing/redisrouter.go:481      could not update node stats     {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).handleRTCMessage
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:481
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:409
2022-11-19T18:25:44.749+0700    ERROR   livekit routing/redisrouter.go:410      error processing RTC message    {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:410
2022-11-19T18:25:46.761+0700    ERROR   livekit routing/redisrouter.go:481      could not update node stats     {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).handleRTCMessage
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:481
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:409
2022-11-19T18:25:46.761+0700    ERROR   livekit routing/redisrouter.go:410      error processing RTC message    {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:410
2022-11-19T18:25:48.766+0700    ERROR   livekit routing/redisrouter.go:481      could not update node stats     {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).handleRTCMessage
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:481
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:409
2022-11-19T18:25:48.766+0700    ERROR   livekit routing/redisrouter.go:410      error processing RTC message    {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.3.1/pkg/routing/redisrouter.go:410
2022-11-19T18:25:50.285+0700    INFO    livekit server/main.go:249      exit requested, shutting down   {"signal": "interrupt"}
2022-11-19T18:25:50.285+0700    DEBUG   livekit routing/redisrouter.go:302      stopping RedisRouter
2022-11-19T18:25:50.286+0700    DEBUG   livekit routing/redisrouter.go:375      finishing redisWorker   {"nodeID": "ND_LIFp2Z23"}

I still see this in v1.4.2 while trying to use ingress with Redis.

```
2023-05-23T16:10:20.856-0500    INFO    livekit redis/redis.go:92       connecting to redis     {"simple": true, "addr": "localhost:6379"}
2023-05-23T16:10:21.028-0500    INFO    livekit service/server.go:219   starting LiveKit server {"portHttp": 7880, "nodeID": "ND_Qltc02Qu", "nodeIP": "172.20.160.1", "version": "1.4.2", "bindAddresses": ["0.0.0.0"], "rtc.portTCP": 7881, "rtc.portUDP": 7882}
2023-05-23T16:10:23.027-0500    ERROR   livekit routing/redisrouter.go:538      could not update node stats     {"error": "loadavg for Windows is not supported"}
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).handleRTCMessage
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.4.2/pkg/routing/redisrouter.go:538
github.com/livekit/livekit-server/pkg/routing.(*RedisRouter).redisWorker
        /home/runner/go/pkg/mod/github.com/livekit/livekit-server@v1.4.2/pkg/routing/redisrouter.go:460
2023-05-23T16:10:23.027-0500    ERROR   livekit routing/redisrouter.go:461      error processing RTC message    {"error": "loadavg for Windows is not supported"
```

--- a/pkg/service/server.go
+++ b/pkg/service/server.go
@@ -8,6 +8,7 @@ import (
 	"net"
 	"net/http"
 	_ "net/http/pprof"
+	"runtime"
 	"runtime/pprof"
 	"time"
 
@@ -217,6 +218,9 @@ func (s *LivekitServer) Start() error {
 		values = append(values, "region", s.config.Region)
 	}
 	logger.Infow("starting LiveKit server", values...)
+	if runtime.GOOS == "windows" {
+		logger.Infow("Windows detected, capacity management is unavailable")
+	}
 
 	for _, promLn := range promListeners {
 		go s.promServer.Serve(promLn)

--- a/pkg/telemetry/prometheus/node.go
+++ b/pkg/telemetry/prometheus/node.go
@@ -3,7 +3,6 @@ package prometheus
 import (
 	"time"
 
-	"github.com/mackerelio/go-osstat/loadavg"
 	"github.com/mackerelio/go-osstat/memory"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/atomic"
@@ -100,7 +99,7 @@ func Init(nodeID string, nodeType livekit.NodeType, env string) {
 }
 
 func GetUpdatedNodeStats(prev *livekit.NodeStats, prevAverage *livekit.NodeStats) (*livekit.NodeStats, bool, error) {
-	loadAvg, err := loadavg.Get()
+	loadAvg, err := getLoadAvg()
 	if err != nil {
 		return nil, false, err
 	}

--- a/pkg/telemetry/prometheus/node_linux.go
+++ b/pkg/telemetry/prometheus/node_linux.go
@@ -5,36 +5,9 @@ package prometheus
 
 import (
 	"fmt"
-	"sync"
 
 	"github.com/florianl/go-tc"
-	"github.com/mackerelio/go-osstat/cpu"
 )
-
-var (
-	cpuStatsLock              sync.RWMutex
-	lastCPUTotal, lastCPUIdle uint64
-)
-
-func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
-	cpuInfo, err := cpu.Get()
-	if err != nil {
-		return
-	}
-
-	cpuStatsLock.Lock()
-	if lastCPUTotal > 0 && lastCPUTotal < cpuInfo.Total {
-		cpuLoad = 1 - float32(cpuInfo.Idle-lastCPUIdle)/float32(cpuInfo.Total-lastCPUTotal)
-	}
-
-	lastCPUTotal = cpuInfo.Total
-	lastCPUIdle = cpuInfo.Idle // + cpu.Iowait
-	cpuStatsLock.Unlock()
-
-	numCPUs = uint32(cpuInfo.CPUCount)
-
-	return
-}
 
 func getTCStats() (packets, drops uint32, err error) {
 	rtnl, err := tc.Open(&tc.Config{})

--- a/pkg/telemetry/prometheus/node_nonlinux.go
+++ b/pkg/telemetry/prometheus/node_nonlinux.go
@@ -2,38 +2,6 @@
 
 package prometheus
 
-import (
-	"runtime"
-	"sync"
-
-	"github.com/mackerelio/go-osstat/cpu"
-)
-
-var (
-	cpuStatsLock              sync.RWMutex
-	lastCPUTotal, lastCPUIdle uint64
-)
-
-func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
-	cpuInfo, err := cpu.Get()
-	if err != nil {
-		return
-	}
-
-	cpuStatsLock.Lock()
-	if lastCPUTotal > 0 && lastCPUTotal < cpuInfo.Total {
-		cpuLoad = 1 - float32(cpuInfo.Idle-lastCPUIdle)/float32(cpuInfo.Total-lastCPUTotal)
-	}
-
-	lastCPUTotal = cpuInfo.Total
-	lastCPUIdle = cpuInfo.Idle
-	cpuStatsLock.Unlock()
-
-	numCPUs = uint32(runtime.NumCPU())
-
-	return
-}
-
 func getTCStats() (packets, drops uint32, err error) {
 	// linux only
 	return

--- a/pkg/telemetry/prometheus/node_nonwindows.go
+++ b/pkg/telemetry/prometheus/node_nonwindows.go
@@ -1,0 +1,56 @@
+//go:build !windows
+
+/*
+ * Copyright 2023 LiveKit, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prometheus
+
+import (
+	"runtime"
+	"sync"
+
+	"github.com/mackerelio/go-osstat/cpu"
+	"github.com/mackerelio/go-osstat/loadavg"
+)
+
+var (
+	cpuStatsLock              sync.RWMutex
+	lastCPUTotal, lastCPUIdle uint64
+)
+
+func getLoadAvg() (*loadavg.Stats, error) {
+	return loadavg.Get()
+}
+
+func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
+	cpuInfo, err := cpu.Get()
+	if err != nil {
+		return
+	}
+
+	cpuStatsLock.Lock()
+	if lastCPUTotal > 0 && lastCPUTotal < cpuInfo.Total {
+		cpuLoad = 1 - float32(cpuInfo.Idle-lastCPUIdle)/float32(cpuInfo.Total-lastCPUTotal)
+	}
+
+	lastCPUTotal = cpuInfo.Total
+	lastCPUIdle = cpuInfo.Idle
+	cpuStatsLock.Unlock()
+
+	numCPUs = uint32(runtime.NumCPU())
+
+	return
+}

--- a/pkg/telemetry/prometheus/node_windows.go
+++ b/pkg/telemetry/prometheus/node_windows.go
@@ -1,0 +1,27 @@
+//go:build !windows
+
+/*
+ * Copyright 2023 LiveKit, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prometheus
+
+func getLoadAvg() (*loadavg.Stats, error) {
+	return &loadavg.Stats{}, nil
+}
+
+func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
+	return 1, 1, nil
+}

--- a/pkg/telemetry/prometheus/node_windows.go
+++ b/pkg/telemetry/prometheus/node_windows.go
@@ -1,4 +1,4 @@
-//go:build !windows
+//go:build windows
 
 /*
  * Copyright 2023 LiveKit, Inc

